proto/osmosis/lockup/lock.proto

```diff
@@ -55,3 +55,14 @@ message QueryCondition {
     (gogoproto.moretags) = "yaml:\"timestamp\""
   ];
 }
+
+// SyntheticLock is a single unit of synthetic lockup
+message SyntheticLock {
+  uint64 lock_id = 1;
```

Comment:
Can this be 'underlying_lock_id' perhaps?

Reply:
Okay, will update to underlying_lock_id

Reply:
Updated